src/solr/mod.rs

```diff
@@ -0,0 +1,58 @@
+use std::collections::HashMap;
+
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "solr";
+const TAG: &str = "9.5.0-slim";
+
+#[derive(Debug, Default)]
+pub struct Solr {
```

Comment:
Could you add some basic documentation like in these examples:
- https://github.com/testcontainers/testcontainers-rs-modules-community/blob/101ccf99f3ed44baab9930af17dbfd1c6bdedbd4/src/rabbitmq/mod.rs#L6-L29
- https://github.com/testcontainers/testcontainers-rs-modules-community/blob/101ccf99f3ed44baab9930af17dbfd1c6bdedbd4/src/surrealdb/mod.rs#L19-L49